src/main/features/claim/views/amount.njk

```diff
@@ -20,13 +20,13 @@
         <div class="claim-amounts">
           <div id="claim-amount">
             <div>
-              <div id="claim-amount-heading-1" class="claim-amount-column-1 mobile-hide">
```

Comment:
Should this class be renamed to phone-hide (it doesn't seem related to phone numbers)?

Reply:
I believe that class is to hide it on mobile devices so it doesn't show when a user is using their phone to view the site